packages/react/index.experimental.development.js

```diff
@@ -29,7 +29,9 @@ export {
   cache,
   cacheSignal,
   startTransition,
-  unstable_Activity,
+  Activity,
+  // TODO: Remove this from experimental after awhile.
+  // Activity as unstable_Activity,
```

Comment:
Is this supposed to be uncommented? `ReactServer.experimental` keeps the prefixed version.

I think it's fine to break `react@experimental` and remove `unstable_Activity`.